packages/react/src/formElements/MdRadioButton.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import classnames from 'classnames';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+export interface MdRadioButtonProps {
+  checked?: boolean | undefined;
+  label?: string;
+  value?: any;
+  id?: string | number;
+  disabled?: boolean;
+  className?: string;
+  onChange(_e: React.ChangeEvent<HTMLInputElement>): void;
+  onBlur?(_e: React.FocusEvent<HTMLInputElement>): void;
+  onFocus?(_e: React.FocusEvent<HTMLInputElement>): void;
+  [otherProps: string]: unknown;
+}
+
+const MdRadioButton: React.FunctionComponent<MdRadioButtonProps> = ({
+  id,
+  disabled,
+  className,
+  value,
+  label,
+  checked,
+  ...otherProps
+}: MdRadioButtonProps) => {
+  const radioGroupId = id || uuidv4();
+  const [,] = useState(false);
+
+  const classNames = classnames(
+    'md-radiobutton',
+    {
+      'md-radiobutton--disabled': !!disabled,
```

Comment:
Consider changing this to an element property and style via `:disabled` instead

Reply:
Evt gjøre disse tingene i ny PR så man slipper å lage ulik konvenskjon på tvers av komponenter. Eks. radiobutton bruker :disabled mens radiogroup bruker --disabled.